File: Linguagem C++/alturas.cpp
#include <bits/stdc++.h>

using namespace std;

int main(){

    int i,n;

    cout << "Quantas pessoas serao digitadas? ";
    cin >> n;
    string nomes[n];
    int idades[n], menorQ16;
    double alturas[n], soma, media,porcentagem;

    for(i = 0; i < n; i++) {
        cout << "Dados da "<< i+1 <<"a pessoa: \n";
        cout << "Nome: ";
        cin.ignore(INT_MAX, '\n');
        getline(cin, nomes[i]);
        cout << "Idade: ";
        cin >> idades[i];
        cout << "Altura: ";
        cin >> alturas[i];
    }

    soma = 0;
    for(i = 0; i < n; i++) {
        soma = soma + alturas[i];
    }
    media = (double)soma / n;
    cout << fixed << setprecision(2);
    cout << "Altura Media = " << media << endl;

    menorQ16 = 0;
    for(i = 0; i < n; i++) {
        if (idades[i] < 16) {
            menorQ16++;
        }
    }
    porcentagem = (double) menorQ16 *100 / n;
    cout << fixed << setprecision(1);
    cout << "Pessoas com menos de 16 anos: " << porcentagem <<"% \n";

    for(i = 0; i < n; i++) {
        if (idades[i] < 16) {
            cout << nomes[i] << endl;
        }
    }





    return 0;
}

File: Linguagem Python/EXs - Vetores PY/alturas.py
n = int(input("Quantas pessoas serao digitadas? "))
nomes: [str] = [0 for x in range(n)]
idades: [int] = [0 for x in range(n)]
alturas: [float] = [0 for x in range(n)]

for i in range(0, n):
    print(f"Dados da {i+1}a pessoa:")
    nomes[i] = input("Nome: ")
    idades[i] = int(input("Idade: "))
    alturas[i] = float(input("Altura: "))

somaAlturas = 0
for i in range(0, n):
    somaAlturas = somaAlturas + alturas[i]

print(f"Altura média = {somaAlturas/n:.2f}")

menoresQ16 = 0
for i in range(0, n):
    if idades[i] < 16:
        menoresQ16 = menoresQ16 + 1

porcentagem = (menoresQ16/n) * 100
print(f"Pessoas com menos de 16 anos: {porcentagem:.1f}%")

for i in range(0, n):
    if idades[i] < 16:
        print(nomes[i])
